cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/config/ETLBatchConfig.java

```diff
@@ -29,34 +31,49 @@
 public final class ETLBatchConfig extends ETLConfig {
   private final String schedule;
   private final List<ETLStage> actions;
+  private final List<LocalizeResourceInfo> resourcesToLocalize;
```

Comment:
It seems that plugins are developed independently, and then composed to a pipeline. How would a plugin developer make use of a localized resource if it can't be configured by the plugin itself? I am not sure whether I understand the use case. Can you give some examples for such resources?